index.html

```diff
@@ -394,8 +394,8 @@ <h3 id="downloads"> Downloads </h3>
     <td><code>https://img.shields.io/chrome-web-store/d/nimelepbpejjlbmoobocpfnjhihnpked.svg</code></td>
   </tr>
   <tr><th data-keywords='website' data-doc='websiteDoc'> Website: </th>
-    <td><img src='https://img.shields.io/website-up-down-green-red/http/shields.io.svg?maxAge=2592000' alt=''/></td>
-    <td><code>https://img.shields.io/website-up-down-green-red/http/shields.io.svg</code></td>
+    <td><img src='https://img.shields.io/website-up-down-green-red-my--website/http/shields.io.svg?maxAge=2592000' alt=''/></td>
+    <td><code>https://img.shields.io/website-up-down-green-red-my--website/http/shields.io.svg</code></td>
```

Comment:
Could you move these changes to `try.html`? Sorry, but this file is autogenerated from that one.